### PR TITLE
[qa] Added dummy model in sample_notifications #45

### LINKED ISSUE #45: [qa] Improving manual testing
Currently, we are relying upon  `openwisp-monitoring` for manually testing new features which are added to this module. We can ease this part by adding a model to the sample app. 

**Idea**
Add a dummy model with name and organisation fields in the sample app. Register a notification type for this model.
 A notification should be generated when objects of this model are created or deleted using this notification type. Notification should be received only by users of the same organisation.  

Thanks to @NoumbissiValere for bringing this up, :+1:

### PR DESCRIPTION
**Pull request by pandafy**
Closes #45

### CODE REVIEW

**Comment by pandafy** [comment]
Tested after rebasing on #44 
Hnet-image (3)

File: tests/openwisp2/sample_notifications/models.py [hunk 1]
```diff
@@ -23,3 +34,25 @@ class NotificationUser(DetailsModel, AbstractNotificationUser):
 
     class Meta(AbstractNotificationUser.Meta):
         abstract = False
+
+
+class OrgUUIDModel(UUIDModel):
```
Comment (NoumbissiValere) [inline tests/openwisp2/sample_notifications/models.py:39]: We could call this `TestApp`

File: tests/openwisp2/sample_notifications/models.py [hunk 2]
```diff
@@ -23,3 +34,25 @@ class NotificationUser(DetailsModel, AbstractNotificationUser):
...
+    organization = models.ForeignKey(
```
Comment (NoumbissiValere) [inline tests/openwisp2/sample_notifications/models.py:40]: Please add also a `name` field.

**Comment by NoumbissiValere** [comment]
@pandafy am glad you are already using it to test. :laughing: this work will greatly ease your development and our testing during the development of next features on our agenda

**Comment by pandafy** [comment]
@NoumbissiValere I have removed generation of notification when object is deleted, since celery task from #44 will delete it without letting us getting glimpse of it.. :smile:

**Pull request merged**